app/models/like.rb

```diff
@@ -1,2 +1,8 @@
 class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def update_post_likes_counter
+    post.update(likes_counter: post.likes.count)
```

Comment:
[OPTIONAL] I think your code will be more concise if you had just done this  post.increment!(:likes_counter). You can also include a after_save callback and make the method private this is a good practice.